### DISCUSSION TITLE
1.7.6 Clarification

### DISCUSSION THREAD

**Discussion by Drewoob** [r57zone/DualShock4-emulator#51] [General]
So i have not noticed the emulation of the thumbsticks, thus, if true, which i doubt, making many games unplayable using the keyboard,  unless i am wrong and blind, which might aswell be the case, i was just wanting to check wether or not I am wrong. I am seeing some LS controls, but i am unsure what those are.

**Comment by r57zone** [comment]
The left stick (LS) is emulated using WASD, and the right stick (RS) is emulated using the mouse.
